src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureDetailer {
+    private String name = "circle";
+    private int radius;
+    private Color color;
```

Comment:
we have `Color color` field in the class Figure, let's use it)
And remove field `private Color color;` from classes: Circle, Square, ...